testing/openmpi/APKBUILD

```diff
@@ -14,13 +14,20 @@ builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
 	cd "$builddir"
+
+	disable_atomics=""
```

Comment:
this should be declared local so it does not conflict with any variable used internally in abuild.
````sh
    local disable_atomics=""
````